docs/source/guides/deployment.rst

```diff
@@ -125,22 +125,35 @@ If they are met, you can move forward with the following command:
 .. code-block:: shell
 
     docker run --name iroha \
-    # External port
     -p 50051:50051 \
-    # Folder with configuration files
     -v ~/Developer/iroha/example:/opt/iroha_data \
-    # Blockstore volume
-    -v blockstore:/tmp/block_store \
-    # Postgres settings
+    -v blockstore:/tmp/block_store \ 
     -e POSTGRES_HOST='some-postgres' \
     -e POSTGRES_PORT='5432' \
     -e POSTGRES_PASSWORD='mysecretpassword' \
     -e POSTGRES_USER='postgres' \
```

Comment:
```suggestion
    -e POSTGRES_USER='postgres' \
     -c 'max_prepared_transactions=100' \
```

Reply:
This is required after #1830

Reply:
Will not work this way. Should be:
```suggestion
     docker run ... hyperledger/iroha:latest -c 'max_prepared_transactions=100'
```

Reply:
Do you mean that is should be mentioned after image name?

Reply:
Yes. This is a Postgres configuration option, not Docker

Reply:
I've rearranged the lines